### PR TITLE
Add support for PHP 8

### PR DESCRIPTION
**Pull request by lepiaf**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
     "type": "library",
     "license": "MIT",
     "require": {
-        "php": "^7.2",
+        "php": ">7.2.5",
```
Comment (stof) [inline composer.json]: why `>` rather than `>=` ?

Comment (lepiaf) [reply composer.json:8]: I did same as https://github.com/swarrot/SwarrotBundle/blob/master/composer.json#L14.

But I can change to 
```
        "php": "^7.2 || ^8.0",
```

Comment (lepiaf) [reply composer.json]: I did same as https://github.com/swarrot/SwarrotBundle/blob/master/composer.json#L14

I will change to `>=` to be same as symfony.

Comment (odolbeau) [reply composer.json]: It fact, as PHP 7.2 is not supported anymore, I'd rather go for `>=7.3`, WDYT? :)

**Comment by odolbeau** [comment]
Thanks @lepiaf!
Could you also update the `.travis.yaml` file please?

**Comment by lepiaf** [comment]
should I update this line in `.travis.yaml` ?

```diff
          # Latest commit to master
-       - php: 7.4
+       - php: 8.0
          env: STABILITY="dev"
```

and remove this change
```
       - php: 8.0
```

**Pull request merged**